tools/uefi_builder/aarch64/Cargo.toml

```diff
@@ -10,7 +10,8 @@ uefi_builder_common = { path = "../common" }
 
 [dependencies.uefi-bootloader]
 artifact = "bin"
-git = "https://github.com/theseus-os/uefi-bootloader"
+git = "https://github.com/NathanRoyer/uefi-bootloader"
+branch = "aarch64-fixes"
```

Comment:
do we still need this? I thought https://github.com/theseus-os/uefi-bootloader/pull/2 fixed this.

Reply:
we don't, sorry